### PR TITLE
increase OI caps from $10k to $1m

### PR DESCRIPTION
**Pull request by cavalier-eth**
pending governance https://github.com/Synthetixio/SIPs/pull/1813

### CODE REVIEW

**Review by noisekit — APPROVED** [review]

**Review by noisekit — CHANGES_REQUESTED** [review]
until approved

File: tomls/omnibus-base-mainnet-andromeda/perps/btc.toml [hunk 1]
```diff
@@ -17,7 +17,7 @@ defaultValue = "0.0002"
 defaultValue = "0.0005"
 
 [setting.perpsBtcMaxMarketSize]
-defaultValue = "0.25" #approx $10k
+defaultValue = "22" #approx $1m
```
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/perps/btc.toml:20]: Need to update tests for btc and eth

Comment (noisekit) [reply tomls/omnibus-base-mainnet-andromeda/perps/btc.toml:20]: tests updated

**Review by noisekit — APPROVED** [review]
Good for me now with tests updated.

File: tomls/omnibus-base-mainnet-andromeda/perps/btc.toml (hunk 1 above)
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/perps/btc.toml:20]: Transactions look good. No unexpected extras

https://app.circleci.com/pipelines/github/Synthetixio/synthetix-deployments/399/workflows/e6c29c44-519f-4360-b00f-1d1f5042af86/jobs/3311/parallel-runs/0/steps/0-106

```sh
Building the chain (ID 8453) via https://mainnet.base.org...
Using 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266

    Downloading ipfs://QmUVbEaFWbjwkJr5wE5cvNzLtybz8QKz4HU7N3GSEV6jsw via ipfs https://us-east.repo.usecannon.com
    Downloading ipfs://QmdkgHhqmD2d5kxddDFj9SAdYBisgZAgh4eQx4ir3a2Yv2 via ipfs https://us-east.repo.usecannon.com
    Downloading ipfs://QmbziafMYsurWLi9PPfKV7dKn35tR4NEJXkgPWVxNdjrp4 via ipfs https://us-east.repo.usecannon.com
    Downloading ipfs://QmUBvYki86Anhzb7Wma6ksZeJtrovUWDSF7aHenEy2w5Cs via ipfs https://us-east.repo.usecannon.com
    Downloading ipfs://QmVCdwqpKQk4QZTP5mw9mPfvvxuH5C34RfW3ZPG3LidX4S via ipfs https://us-east.repo.usecannon.com
    Downloading ipfs://QmSMDL9NiVjCky5Tu71ZYyMeBe9Fn3NbyRQCbkUemqbfPt via ipfs https://us-east.repo.usecannon.com
Executing [invoke.setPerpsBtcMaxMarketSize]...
  ✔ Successfully called setMaxMarketSize(200, 22000000000000000000)
  Signer: 0xbb63CA5554dc4CcaCa4EDd6ECC2837d5EFe83C82
  Contract Address: 0x0A2AF931eFFd34b81ebcc57E3d3c9B1E1dE1C9Ce
  Transaction Hash: 0xb44c742fe05670d0c9b8249a11f8eb2594a61e1fc102bce79493fe2d3e38d64f
  Transaction Cost: 0.000000000009522151 ETH (39,511 gas)

Executing [invoke.setPerpsEthMaxMarketSize]...
  ✔ Successfully called setMaxMarketSize(100, 385000000000000000000)
  Signer: 0xbb63CA5554dc4CcaCa4EDd6ECC2837d5EFe83C82
  Contract Address: 0x0A2AF931eFFd34b81ebcc57E3d3c9B1E1dE1C9Ce
  Transaction Hash: 0xa84ca29666eb67b9d2d390c60d9a638730f6e7d1a83ff2ca7a4c3d9f0d6f7190
  Transaction Cost: 0.000000000008336821 ETH (39,511 gas)

💥 synthetix-omnibus:3.3.5-2@andromeda built on Base Mainnet (Chain ID: 8453)
Total Cost: 0.000000000017858972 ETH
```

**Comment by cavalier-eth** [comment]
approved https://github.com/Synthetixio/SIPs/pull/1823

**Pull request merged**